services/normalDocSvc.js

```diff
@@ -153,13 +153,26 @@ angular.module('o19s.splainer-search')
       return doc;
     };
 
+    var getSolrDocExplain = function(solrDoc, nDoc) {
+      var explJson = solrDoc.explain(nDoc.id);
+      if (explJson === null) {
+        if (solrDoc.source().hasOwnProperty('id')) {
+          console.log('for id');
```

Comment:
why leave the console log statements in?  Are you expecting users to need these somehow?